README.md

```diff
@@ -75,4 +76,19 @@ for testing:
 #### Accuracy:
 - Consider that pixel in item mask could be addressed with area (set of pixels) in result mask
 - replace naive algorithm of placing with the arranging of objects by parallel transfer
-- use physics solutions: place objects randomly to polygon and use potential field method
\ No newline at end of file
+- use physics solutions: place objects randomly to polygon and use potential field method
+
+## Improved in version 2
+- implemented new method od placing objects: objects are placed recursively with movements and rotations by 5 degree per iteration
+- modified classification method: now contours get less impact and descriptors - more, calculated distances of pairs 
+and only "good" pairs are considered 
+- added exceptional check for red circle and blue phone(rectangle)
+- resized input images so algorithm works faster
+- started implementation of genetic algo for placing problem
+- extended dataset (added more examples with rotations and included example with shadow impact where algorithm don't work)
+
+## Improvements todo from version 2
+- add the ability to parallelize evaluations in brute force parallel-rotate solution
+- implement genetic algo
+- add exceptional check for keychain pass
+- evaluate precicion, accuracy and recall automatically (only manual for now)
```

Comment:
I think, it is also necessary to describe the metrics specified in the .ipynb-file here